web/themes/custom/cloudy/src/css/_print.scss

```diff
@@ -0,0 +1,15 @@
+/*
+** Print-Specific Styles
+*/
+
+@media print {
+
+  // Targets all FireFox browsers
+  @-moz-document url-prefix() {
```

Comment:
Wow... I had hoped it would be this simple. Thanks for tracking that down @rockymountainhigh1943 !